src/gui/sharee.cpp

```diff
@@ -100,6 +100,11 @@ void ShareeModel::shareesFetched(const QJsonDocument &reply)
             newSharees.append(parseSharee(group.toObject()));
         }
 
+        auto emails = exact.value("emails").toArray();
+        foreach (auto email, emails) {
```

Comment:
I'd have things to say about that foreach but let's keep it for now. ;-)

It'll go away the day I finally find the time to make use of clazy on the CI.